recipes/configs/llama2/13B_full.yaml

```diff
@@ -83,3 +83,7 @@ metric_logger:
   log_dir: ${output_dir}
 output_dir: /tmp/alpaca-llama2-finetune
 log_every_n_steps: null
+
+# Pytorch Profiling
+enable_torch_profiler: False
+output_file_path: ./torchtune_perf_tracing.json
```

Comment:
Not unique to this change but I feel we are starting to accumulate too many potentially disparate file paths here (logger, checkpoint output files, now profiling file path). As of now none of these are aware of each other, but I don't see why we shouldn't just have a single output directory for all this stuff.